src/main/java/scala/compat/java8/wrappers/IteratorPrimitiveDoubleWrapper.java

```diff
@@ -0,0 +1,13 @@
+package scala.compat.java8.wrappers;
```

Comment:
I was hoping to keep all API from this module under `scala.compat`, rather than trying to augment standard library packages.

The basis of this decision was originally to avoid shipping "split packages". Thinking about it some more, this might be less of a problem if we just add a package in a stdlib package, rather than directly add classes.

Here's the original discussion: https://groups.google.com/a/typesafe.com/forum/#!search/scala-java8-fun/eng/iUyQxzcsGfs/3jvl6cfTYhoJ